script/src/verify.rs

```diff
@@ -761,97 +767,21 @@ mod tests {
         );
     }
 
-    #[test]
```

Comment:
Why remove it?

Reply:
This unit test case is trying to run a script with low cycle, it's covered in `check_signature` line 600